.github/workflows/publish.yml

```diff
@@ -31,6 +31,9 @@ jobs:
         with:
           inherit-toolchain: true
```

Comment:
Any concern we just remove `inherit-toolchain`? It should automatically pick the current stable version IIRC. Otherwise this becomes another place we must remember to update once in a while.

Reply:
Oh that works better, good point